<ui version="4.0" >
 <class>MainWindowBase</class>
 <widget class="QMainWindow" name="MainWindowBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string comment="@title:window" >KDE Partition Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="PartTableWidget" native="1" name="m_PartTableWidget" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::CustomContextMenu</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="m_TreePartitions" >
        <property name="contextMenuPolicy" >
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="itemsExpandable" >
         <bool>false</bool>
        </property>
        <column>
         <property name="text" >
          <string comment="@label">Partition</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string comment="@label partition type">Type</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string comment="@label">Mount point</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string comment="@label file system label">Label</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string comment="@label total file system size">Size</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string comment="@label space used">Used</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string comment="@label partition flags">Flags</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="m_DockDevices" >
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string comment="@title:window" >Devices</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="m_DockDevicesContents" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QListWidget" name="m_ListDevices" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy" >
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="m_DockOperations" >
   <property name="allowedAreas" >
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string comment="@title:window" >Pending Operations</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="m_DockOperationsContents" >
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QListWidget" name="m_ListOperations" >
       <property name="horizontalScrollBarPolicy" >
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScroll" >
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="resizeMode" >
        <enum>QListView::Adjust</enum>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>m_ListOperations</zorder>
   </widget>
  </widget>
  <widget class="QDockWidget" name="m_DockInformation" >
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string comment="@title:window" >Information</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="m_DockInformationContents" >
    <layout class="QGridLayout" name="_2" />
   </widget>
  </widget>
  <widget class="QDockWidget" name="m_DockLog" >
   <property name="allowedAreas" >
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string comment="@title:window" >Log Output</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QTreeWidget" name="m_TreeLog" >
       <property name="autoScroll" >
        <bool>false</bool>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="textElideMode" >
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="indentation" >
        <number>0</number>
       </property>
       <property name="rootIsDecorated" >
        <bool>false</bool>
       </property>
       <property name="itemsExpandable" >
        <bool>false</bool>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
       <property name="headerHidden" >
        <bool>true</bool>
       </property>
       <property name="expandsOnDoubleClick" >
        <bool>false</bool>
       </property>
       <column>
        <property name="text" >
         <string comment="@label">Time</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string comment="@label">Message</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PartTableWidget</class>
   <extends>QWidget</extends>
   <header>gui/parttablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
